plottr/data/qcodes_dataset.py

```diff
@@ -203,31 +203,34 @@ class QCodesDSLoader(Node):
     uiClass = None
     useUi = False
 
-    def __init__(self, *arg, **kw):
-        self._pathAndId = (None, None)
+    def __init__(self, *arg: Any, **kw: Any):
+        self._pathAndId: Tuple[Optional[str], Optional[int]] = (None, None)
         self.nLoadedRecords = 0
 
         super().__init__(*arg, **kw)
 
     ### Properties
 
     @property
-    def pathAndId(self):
+    def pathAndId(self) -> Tuple[Optional[str], Optional[int]]:
         return self._pathAndId
 
     # see https://github.com/python/mypy/issues/1362
     @pathAndId.setter  # type: ignore
     @updateOption('pathAndId')
-    def pathAndId(self, val):
+    def pathAndId(self, val: Tuple[Optional[str], Optional[int]]) -> None:
         if val != self.pathAndId:
             self._pathAndId = val
             self.nLoadedRecords = 0
 
     ### processing
-
-    def process(self, **kw):
+    # mypy does not correctly see the return annotation of pathAndId due to lack of support
+    # for decorated properties
```

Comment:
should this comment be on line 218?

Reply:
Turns out this was related to the other one below